docs/pages/deploy/security.md

```
@@ -25,57 +25,92 @@ Open this URL in your browser:
+Once logged in, all relevant [CLI commands](../reference/cli.md) use the saved credentials automatically:
 
 ```console
-$ statespace deploy ./myapp
+$ statespace deploy <PATH>
 $ statespace app list
-$ statespace app delete <app-id>
+$ statespace app delete <APP>
 ```
 
-Alternatively, set the `STATESPACE_API_KEY` environment variable:
+Alternatively, pass an API key directly with `--api-key`:
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)